src/components/course/data/utils.jsx

```diff
@@ -463,3 +588,70 @@ export const getCourseRunPrice = ({
  * @returns Casts value to a float and fixes it to 2 decimal places.
  */
 export const fixDecimalNumber = (value) => parseFloat(value).toFixed(2);
+
+/**
+ * TODO
+ */
+export const getMissingSubsidyReasonActions = ({
+  reasonType,
+  enterpriseAdminUsers,
+}) => {
+  const hasLimitsLearnMoreCTA = [
+    DISABLED_ENROLL_REASON_TYPES.LEARNER_MAX_SPEND_REACHED,
+    DISABLED_ENROLL_REASON_TYPES.LEARNER_MAX_ENROLLMENTS_REACHED,
+  ].includes(reasonType);
+  const hasOrganizationNoFundsCTA = [
+    DISABLED_ENROLL_REASON_TYPES.NO_SUBSIDY,
+    DISABLED_ENROLL_REASON_TYPES.POLICY_NOT_ACTIVE,
+    DISABLED_ENROLL_REASON_TYPES.LEARNER_NOT_IN_ENTERPRISE,
+    DISABLED_ENROLL_REASON_TYPES.CONTENT_NOT_IN_CATALOG,
+    DISABLED_ENROLL_REASON_TYPES.NOT_ENOUGH_VALUE_IN_SUBSIDY,
+  ].includes(reasonType);
+
+  console.log('getMissingSubsidyReasonActions', {
+    hasLimitsLearnMoreCTA,
+    hasOrganizationNoFundsCTA,
+    enterpriseAdminUsers,
+    reasonType,
+  });
+
+  if (hasLimitsLearnMoreCTA) {
+    return (
+      <Button
+        as={Hyperlink}
+        // TODO: we don't yet have a destination for this link
```

Comment:
We do now! https://business-support.edx.org/hc/en-us/articles/14473635245079-Learner-Spending-Limits

Reply:
Updated.